File: app/src/main/java/com/example/administrator/essim/activities/UserDetailActivity.java
package com.example.administrator.essim.activities;

import android.content.Intent;
import android.graphics.Color;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.ProgressBar;

import com.example.administrator.essim.R;


public class UserDetailActivity extends AppCompatActivity {

    public int userId;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getWindow().setStatusBarColor(Color.TRANSPARENT);
        getWindow().getDecorView().setSystemUiVisibility(View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN |
                View.SYSTEM_UI_FLAG_LAYOUT_STABLE);
        Intent intent = getIntent();
        userId = intent.getIntExtra("user id", 0);
        setContentView(R.layout.activity_myself);
    }
}

File: app/src/main/java/com/example/administrator/essim/activities/UserDetailActivity.kt
package com.example.administrator.essim.activities

import android.graphics.Color
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View
import com.example.administrator.essim.R


class UserDetailActivity : AppCompatActivity() {

    var userId: Int = 0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        window.statusBarColor = Color.TRANSPARENT
        window.decorView.systemUiVisibility = View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN or View.SYSTEM_UI_FLAG_LAYOUT_STABLE
        val intent = intent
        userId = intent.getIntExtra("user id", 0)
        setContentView(R.layout.activity_myself)
    }
}
